CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 1.0.0 (Unreleased)
```

Comment:
The `(Unreleased)` string needs to remain in place. It's used as a marker buy the release automation and will be replaced with the actual release date.